crates/re_viewer/src/misc/mesh_loader.rs

```diff
@@ -121,7 +119,7 @@ impl LoadedMesh {
         let vertex_colors = if let Some(vertex_colors) = vertex_colors {
             vertex_colors
                 .iter()
-                .map(|c| Rgba32Unmul::from_rgba_unmul_array(c.to_array()))
+                .map(|c| Rgba32Unmul::from_rgba_unmul_array(ColorRGBA(*c).to_array()))
```

Comment:
`c` is not already an array/slice that can be passed directly?

Reply:
I think we're basically doing an endianess swap here.

ColorRGBA stores the color as 0xRRGGBBAA, but to_array converts it to [r,g,b,a].
```
    pub fn to_array(&self) -> [u8; 4] {
        [
            (self.0 >> 24) as u8,
            (self.0 >> 16) as u8,
            (self.0 >> 8) as u8,
            self.0 as u8,
        ]
    }
```

Reply:
it's a matter of turning a `u32` into a `Rgba32Unmul`. We could add a `Rgba32Unmul::from_u32` helper but then we would need to document exactly how that works etc…